Jenkinsfile

```diff
@@ -8,11 +8,11 @@ def sourceRepo
 def artifactId
 def pipelineMetadata = [
     pipelineName: 'dist-git',
-    pipelineDescription: 'Run tier-0 tests from dist-git',
-    testCategory: 'functional',
-    testType: 'tier0',
+    pipelineDescription: 'create an scratch build from PR',
+    testCategory: 'static-analysis',
+    testType: 'build',
```

Comment:
What about:
```
    testCategory: 'scratch-build',
    testType: 'koji',
```
?

Reply:
I think the only possible categories are https://pagure.io/fedora-ci/messages/blob/master/f/schemas/test-common.yaml#_12

type I could change to `koji`, but `static-analysis` and `build` is what we use downstream...

Reply:
Damn. You're right. Ok, let's keep it as it is for now then :+1: 

I am not sure if people will be able to connect the dots and realize that build.static-analysis is just a scratch-build, but we can improve this later.

Reply:
actually, what the user will see if the pagure flag info, this will be set by toddlers: https://pagure.io/fedora-infra/toddlers/pull-request/64#_2__26